source/slang/core.meta.slang

```diff
@@ -1720,6 +1720,57 @@ T __attachToNativeRef(NativeRef<T> nativeVal)
 }
 
 //@public:
+__generic<T, let N:int>
+__magic_type(ArrayExpressionType)
+struct Array : IRWArray<T>
+{
+    __intrinsic_op($(kIROp_GetArrayLength))
+    int getCount();
+}
+
+//@public:
+__generic<int N>
+__magic_type(StringLiteralExpressionType)
+__intrinsic_type($(kIROp_StringLiteralType))
+struct StringLiteral : IArray<uint>
+{
+    __intrinsic_op($(kIROp_GetStringLiteralLength))
+    [__unsafeForceInlineEarly]
+    int getCount();
```

Comment:
why do we need both `getCount` and `length`? I would prefer we just have `length`.